torch/__init__.py

```diff
@@ -769,6 +770,11 @@ def compiled_with_cxx11_abi():
 
 from torch.utils.dlpack import from_dlpack, to_dlpack
 
+# this was originally defined in torch.functional and made available as
+# torch.tensordot. Now is has been moved to torch.linalg.tensordot, but we
+# keep it available in the old namespaces to preserve backward compatibility.
+tensordot = linalg.tensordot
```

Comment:
Can't the functional tensordot stick around and just call the function it always has though instead of changing __init__.py?